Dockerfile

```diff
@@ -2,11 +2,13 @@
 FROM python:3.9
 
 # Set working directory.
-
+WORKDIR /fastapi
+COPY ./requirements.txt /fastapi/requirements.txt
 # Expose port 8000 for your FastAPI server.
-
+EXPOSE 8000
 # Install Python requirements using Pip.
-
+RUN pip install -r requirements.txt
 # Copy your codebase into the Docker container.
-
+COPY ./* /fastapi/
 # Run FastAPI server on the port exposed above.
+CMD [ "uvicorn" ,"main:app",  "--host", "0.0.0.0", "--reload" ]
```

Comment:
One other point: there's a flag for `uvicorn` that enables you to set the port your server will be running on. By default, it's 8000, so it works in this case without having set that flag. But keep this in mind, in case you ever run into a situation where you need to run on a different port.